code-challenges.test.js

```diff
@@ -27,26 +45,74 @@ var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saf
 
 // b) Create the function that makes the test pass.
 
+const shuffleMe = (arr) =>{
+    let shiftedArr = arr.slice(1)
+    for( let i = shiftedArr.length-1; i>0; i--){
+        let j = Math.floor( Math.random()* (i+1));
+        [shiftedArr[i],shiftedArr[j]] = [shiftedArr[j], shiftedArr[i]];
+    }
+    return shiftedArr
+}
+console.log(shuffleMe(colors1))
+
+describe("The shuffleMe function.",()=>{
+    test("takes an array and returns a new array with the first index of the old array removed and a randomized order",() => {
+        expect(shuffleMe(colors1).sort()).toEqual(["blue", "green","pink", "yellow"])
+        expect(shuffleMe(colors1)).not.toEqual(colors1)
+        expect(shuffleMe(colors2).sort()).toEqual(["aquamarine", "indigo","ochre","periwinkle",  "saffron"])
+        expect(shuffleMe(colors2)).not.toEqual(colors2)
+
+
+    })
+})
```

Comment:
Good stuff! I am assuming you wrote the test first and watched it fail then wrote the code above it...